text/chapter7.md

```diff
@@ -674,18 +674,19 @@ In the case of an empty list, we can simply return an empty list using `pure`. I
 
 But there are more examples of traversable functors than just arrays and lists. The `Maybe` type constructor we saw earlier also has an instance for `Traversable`. We can try it in PSCi:
 
-```text
+```haskell
```

Comment:
All repl blocks use `text` formatting. Not against changing it, but it should be done for the book as a whole.

Reply:
@milesfrain yep, you're right.  I _think_ it might be helpful to have the code formatting in the repl blocks; it looked good when I did it, but I'm not invested in it enough to do the required "due diligence" right now to make sure it's correct.

I'm not sure it's even worth adding an issue on it right now.

Thanks for catching it.